citesphere/src/main/java/edu/asu/diging/citesphere/core/zotero/impl/ZoteroConnector.java

```diff
@@ -242,11 +244,27 @@ private Zotero getApi(IUser user) {
         zotero.setUserId(token.getUserId());
         return zotero;
     }
-    
+
     private ZoteroHttpStatusException createException(HttpStatus status, Exception cause) {
         if (status == HttpStatus.FORBIDDEN) {
             return new AccessForbiddenException(cause);
         }
         return new ZoteroHttpStatusException(cause);
     }
+
+    @Override
+    public void deleteItem(IUser user, String groupId, String citationKey, Long citationVersion)
+            throws ZoteroConnectionException, ZoteroHttpStatusException, ZoteroItemDeletionFailedException {
+        Zotero zotero = getApi(user);
+        if (!zotero.getGroupsOperations().deleteItem(groupId, citationKey, citationVersion)) {
+            throw new ZoteroItemDeletionFailedException("Deletion of Citation failed");
```

Comment:
what's the difference between ZoteroItemDeletionFailedException and ZoteroHttpStatusException?